pages/community/community-support-precice.md

```diff
@@ -0,0 +1,52 @@
+---
+title: You support preCICE - preCICE supports you
+permalink: community-support-precice.html
+keywords:
+summary:
+toc: true
+editme: false
+---
+
+## Why is support for preCICE necessary?
+
+Unfortunately the majority of open source software is abandoned at some point, because the developers lack the resources to maintain and develop the software. Using open source software in a professional context can therefore be considered risky, because the long-term sustainability is uncertain.  In order to ensure a sustainable future for preCICE, we are taking steps  to make sure preCICE remains the leading, state-of-the-art coupling solution. By supporting preCICE as an academic or industry user, you contribute to the long-term development of preCICE and will receive dedicated support from the preCICE development team.
+
+## Funding and support options
+
+There are three different ways to fund and support the development of preCICE.
+
+### 1. Research project partner
+
+Include preCICE as a project partner in your research project proposal or team up with the preCICE developers to write a proposal. This is the best option when you want to benefit from the research and scientific expertise of the preCICE team, and increase the chances that your project is funded. The developers of preCICE are experts in state-of-the-art numerical methods, high-performance computing, and software engineering for multi-X coupled problems. Reviewers appreciate the highly recognized expertise of preCICE, demonstrated by our fast growing [number of citations](https://scholar.google.com/scholar?hl=en&cites=5053469347483527186), and [vibrant community](community-projects.html).
+
+Directly [contact Jun. Prof. Benjamin Uekermann](https://www.ipvs.uni-stuttgart.de/departments/us3/) to discuss a collaboration.
+
+### 2. Support license
```

Comment:
Harald gave feedback that the word "license" might make things more complicated in an industrial environment as it might mean that the billing needs to go through the central software purchasing department. The obvious alternative would be "support contract". On the other hand, I think that the wording "license" could make it easier in an academic environment. Other opinions?

Reply:
That is a good point. Generally speaking everything a company above a certain size buys needs to go through purchasing as the default process. Also, very generally speaking, there will be exceptions and we should aim for these, but be prepared for the default.

What preCICE support offers is a service, however (independent of what the service is called), and it is hard to predict who the actual stakeholders in the purchasing process will be. I would keep license for now, and adapt when we know more.